.claude/skills/changelog-collect/SKILL.md

```
@@ -0,0 +1,328 @@
+
+返回字段：
+
+- `title`: PR 标题
+- `body`: PR 描述（包含中英文 changelog）
+- `author`: 提交者
+
+#### 1.5 从 PR body 提取中英文描述
+
+PR body 中提取 changelog 的模式：
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)